### PR TITLE
[IE CLDNN] Eltwise + quantize fusing for ref and fsv16 kernels 

### PR DESCRIPTION
**Pull request by cepera**
CVS-27996: support added for ref and b_fs_yx_fsv16 implementations/
PS this one is a copy of #673. had to create a new one because #673 is broken because of LFS files

### CODE REVIEW

File: inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/eltwise/eltwise_kernel_ref.cpp [hunk 1]
```diff
@@ -54,4 +54,24 @@ bool EltwiseKernelRef::Validate(const Params& p, const optional_params& o) const
...
+
+    if (!params.fused_ops.empty()) {
+        kernel_selector::Datatype input_dt = GetAccumulatorType(params);
+
+        std::vector<std::string> idx_order;
+        if (DataTensor::ChannelsCount(params.output.GetLayout()) == 4) {
+            idx_order = {"d4", "d3", "d2", "d1"};
+        } else if (DataTensor::ChannelsCount(params.output.GetLayout()) == 5) {
+            idx_order = {"d5", "d4", "d3", "d2", "d1"};
+        }
```
Comment (sshlyapn) [inline inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/eltwise/eltwise_kernel_ref.cpp:69]: Reference kernel also supports 6D layouts, so **bfwzyx** case should be handled too

File: inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/eltwise_b_fs_yx_fsv16.cl [hunk 2]
```diff
@@ -15,21 +15,30 @@
...
-#include "include/unit_type.cl"
+#include "include/common.cl"
+#include "include/data_types.cl"
 
 #define FEATURE_SLICE_SIZE 16
 
+#define OUTPUT_TYPE_BLOCK               MAKE_VECTOR_TYPE(OUTPUT_TYPE, BLOCK_SIZE)
+#define TO_OUTPUT_TYPE_BLOCK(val)       CAT(convert_, OUTPUT_TYPE_BLOCK)(val)
+
+
```
Comment (sshlyapn) [inline inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/eltwise_b_fs_yx_fsv16.cl:26]: Please delete one empty line

File: inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/eltwise/eltwise_kernel_b_fs_yx_fsv16.cpp [hunk 3]
```diff
@@ -163,6 +176,9 @@ bool EltwiseKernel_b_fs_yx_fsv16::Validate(const Params& params, const optional_
 
     auto input0 = ewParams.inputs[0];
 
+    if (ewParams.inputs.size() == 2 && input0.GetDType() != ewParams.inputs[1].GetDType())
+        return false;
```
[the remark this answers is not in the document]
Comment (cepera) [reply inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/eltwise/eltwise_kernel_b_fs_yx_fsv16.cpp]: makes sense, thanks! changed.

**Comment by vladimir-paramuzov** [comment]
@cepera Could you rebase on the latest master? I see some strange regression in validation results, so I'd want to restart it and double check the report.

**Comment by cepera** [comment]
> @cepera Could you rebase on the latest master? I see some strange regression in validation results, so I'd want to restart it and double check the report.

done. Can you please recheck?

**Review by vladimir-paramuzov — APPROVED** [review]

**Pull request merged**